content/kubermatic/main/tutorials-howtos/kubeone-clusters/_index.en.md

```diff
@@ -0,0 +1,105 @@
+
+  Currently Supported Providers:
+   - [AWS]({{< ref "./aws" >}})
+   - [Google Cloud Provider]({{< ref "./gcp" >}})
+   - [Azure]({{< ref "./azure" >}})
+
+## Prerequisites
+
+The following requirements must be met to import a KubeOne cluster:
+ - The KubeOne cluster must already exist before we begin the import/connect process.
+ - Manifest: KubeOne can define our KubeOne clusters declaratively, in a form of a YAML manifest.
+   Run `kubeone config dump -m kubeone.yaml -t tf.json` in our KubeOne terraform directory to get a manifest.
+ - Private SSH Key used to create the KubeOne cluster:  KubeOne connects to instances over SSH to perform any management   operation.
+ - Provider Specific Credentials used to create the cluster.
+
+ > For more information on the kubeone configurations for different environment, checkout the [Creating the kubernetes Cluster using Kubeone]({{< relref "../../../../kubeone/main/tutorials/creating-clusters/" >}}) documentation.
+
+## Import KubeOne Cluster
+
+KKP allows connecting any existing KubeOne cluster of supported provider to view and edit the cluster's current state.
+
+- To import a KubeOne cluster go to `KubeOne Clusters` page and Click the `Import KubeOne Cluster` button.
+
+![Import KubeOne Cluster](/img/kubermatic/main/tutorials/kubeone_clusters/cluster_list_empty.png "Import KubeOne Cluster")
+
+- Select the KubeOne cloud provider.
+
+![Select Provider](/img/kubermatic/main/tutorials/kubeone_clusters/import_kubeone_cluster.png "Select Provider")
+
+
+We can then see the details of the cluster.
+
+![Cluster Details](/img/kubermatic/main/tutorials/kubeone_clusters/cluster_details.png "Imported AWS Cluster")
+
+![KubeOne Cluster List](/img/kubermatic/main/tutorials/kubeone_clusters/cluster_list.png "KubeOne Cluster List")
+
+## Cluster Details Page
+
+After the cluster is imported, the KKP controller retrieves the cluster kubeconfig to display all necessary information.
+A healthy cluster has `Running` state. Move the mouse cursor over the state indicator to get more details.
+
+## Update Cluster
+
+> KKP supports latest KubeOne code hence supported kubernetes version '>= 1.24'.
+We need to use an older KubeOne version to upgrade ou cluster to a supported version using KubeOne cli.
+Please refer to the [KubeOne Compatibility section]({{< relref "../../../../kubeone/main/architecture/compatibility/supported-versions/" >}}) docs for more details.
+
+### Upgrade Cluster Version
+
+When an upgrade for the cluster is available, a little dropdown arrow will be shown beside the `Control Plane Version` on the cluster’s page.
+To start the upgrade, choose the desired version from the list of available upgrade versions and click on `Change Version`.
+
+![Upgrade Cluster](/img/kubermatic/main/tutorials/kubeone_clusters/upgrade_cluster.png "Upgrade Cluster")
+
+If the version upgrade is valid, the cluster state will change to `Reconciling`.
+
+## Update the Machine Deployment Version
```

Comment:
I would add some intro, such as: "Kubelet version on the worker nodes managed by Machine Deployments can be updated as follows:"